package.json

```diff
@@ -56,6 +56,6 @@
   },
   "devDependencies": {},
   "engines": {
-    "node": ">=6.9.2"
+    "node": ">=10"
```

Comment:
RE: the previous question, I'm *hoping* we can keep this 8.x LTS+... I'm just not sure yet.